### PR TITLE
Migrate to RTL (Dropdown, ToastHeader)

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [ ] Bug fix 
- [ ] New feature 
- [x] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to [Typescript typings](./types/lib).
  - [ ] I have updated the typings accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/__tests__/ToastHeader.spec.js [hunk 1]
```diff
@@ -1,44 +1,30 @@
...
-    expect(wrapper.find('button.btn-close').length).toBe(1);
+    expect(screen.getByLabelText(/close/i)).toBeInTheDocument();
   });
 
   it('should render custom tag', () => {
-    const wrapper = shallow(<ToastHeader tag="p">Yo!</ToastHeader>).childAt(0);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.type()).toBe('p');
+    render(<ToastHeader tag="p">Yo!</ToastHeader>);
```
Comment (davidacevedo) [inline src/__tests__/ToastHeader.spec.js:22]: I believe you made a `testForCustomTag` utility function. So maybe use that?

File: src/__tests__/Dropdown.spec.js [hunk 2]
```diff
@@ -1,388 +1,320 @@
...
             <DropdownItem>Test</DropdownItem>
           </DropdownMenu>
         </Dropdown>,
       );
 
-      expect(wrapper.find('.dropdown').prop('inNavbar')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('toggle')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('a11y')).toBe(undefined);
-      expect(wrapper.find('.dropdown').prop('isOpen')).toBe(undefined);
+      const dropdown = document.getElementsByClassName('dropdown')[0];
```
Comment (davidacevedo) [inline src/__tests__/Dropdown.spec.js:103]: Not 100%, but you might be able to get this element from `const { container } = render(...)` where container is dropdown

**Review by davidacevedo — APPROVED** [review]
This test was a pretty good read 🥳

File: src/__tests__/ToastHeader.spec.js (hunk 1 above)
Comment (illiteratewriter) [reply src/__tests__/ToastHeader.spec.js:22]: `TestHeader` works a bit differently. The `prop` name for changing the tag for external class in `ToastHeader` is `wrapTag` and not `tag`. So can't use that function here.

File: src/__tests__/Dropdown.spec.js (hunk 2 above)
Comment (illiteratewriter) [reply src/__tests__/Dropdown.spec.js:103]: I think `container` returns a `div` above the `Dropdown` and not dropdown itself. And it is a fiber node and not a DOM node. I'm also not sure tbh, but was unable to make it work with `container`

Something like 
```
<body>
  <div> // container is this
    <Dropdown />
  </div>
<body>
```

Comment (davidacevedo) [reply src/__tests__/Dropdown.spec.js:103]: I see, thanks for checking that out!

File: src/__tests__/ToastHeader.spec.js (hunk 1 above)
Comment (davidacevedo) [reply src/__tests__/ToastHeader.spec.js:22]: Makes sense and sounds good!

**Pull request merged**